pyrightconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "include": ["ddtrace"],
+  "exclude": [
+    "**/node_modules",
+    "**/__pycache__",
+    ".git",
+    "ddtrace/vendor",
+    "ddtrace/contrib"
+  ],
+  "executionEnvironments": [
+    {
+      "root": "ddtrace"
```

Comment:
I think we want this as `.` instead of `ddtrace`

Reply:
This resolved the import issues